src/main/java/Storage.java

```diff
@@ -0,0 +1,67 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    public static String DEFAULT_PATH = "./data.txt";
+
+    public static void ensureFileExists() {
+        File file = new File(DEFAULT_PATH);
+        if (file.exists()) return;
```

Comment:
Hi, according to the module's coding standard, single statement conditionals should still be wrapped by curly brackets